apps/teams-test-app/src/pages/TestApp.tsx

```diff
@@ -123,6 +124,7 @@ export const TestApp: React.FC = () => {
         <SharingAPIs />
         <WebStorageAPIs />
         <StageViewAPIs />
+        <StageViewSelfAPIs />
```

Comment:
Could you please sort it back to alphabetical order next time when you write E2E test please?